tinyagentos/routes/library.py

```
@@ -0,0 +1,274 @@
+        )
+    else:
+        return JSONResponse(
+            {"error": "Provide either 'file' (multipart upload) or 'url' (form field)."},
+            status_code=400,
+        )
+
+    # Run pipeline in background
+    task_set = getattr(request.app.state, "_background_tasks", None)
+    coro = _ingest_task(request.app, item_id, store, storage_dir)
+    if task_set is None:
+        asyncio.create_task(coro)
```

Ruff (0.15.21)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)